packages/jest-jasmine2/src/treeProcessor.ts

```diff
@@ -78,7 +78,7 @@ export default function treeProcessor(options: Options): void {
     const children = node.children.map(child => ({
       fn: getNodeHandler(child, enabled),
     }));
-    if (!hasEnabledTest(node)) {
+    if (!hasEnabledTest(node) || node.markedPending) {
```

Comment:
should `hasEnabledTests` check `markedPending`? No biggie of course

Reply:
It was actually a biggie. I added more tests. Please review ᅟagain.